src/server/models/cacheable_queries/user.js

```diff
@@ -32,6 +32,55 @@ export async function userHasRole(userId, orgId, acceptableRoles) {
   }
 }
 
+async function getUserOrgInfo(userId, orgId) {
```

Comment:
I think this does two separate things
1. tests membership
2. gets org name and caches it.

For (1) it is already cached in `texterinfo-<userid>` with key '<orgId>' so we shouldn't recache it again -- and just use that.
For (2), we don't need to save that per-user, because it's at the org level.  and that should be able to use the general org cache in this PR: https://github.com/MoveOnOrg/Spoke/pull/855/files#diff-a384a0e3111ace4e3ecc640befd115e4R11

I know in some cases we just want the org name for the user query -- in that case, maybe worth stuffing it into the already-existing `texterinfo-<userid>` and maybe split role:orgName or something.